app/models/view/daily_calculations.rb

```diff
@@ -34,42 +34,39 @@ def reserved_rate_status
       RatioPresenter.new(reserved_ratio).alert_status
     end
 
-    private
-
-    class RatioPresenter
-      attr_reader :number
-
-      def initialize(number)
-        @number = number
-      end
-
-      def percentage
-        "#{percentile}%"
-      end
+    def seven_day_discharge_average_per_hundred
+      AveragePresenter.new(seven_day_discharge_average).average_per_hundred
+    end
 
-      def alert_status
-        if percentile <=65
-          'notice'
-        elsif percentile <= 85
-          'warn'
-        else
-          'alert'
-        end
-      end
+    def seven_day_discharge_average_per_hundred_status
+      AveragePresenter.new(seven_day_discharge_average).alert_status
+    end
 
-      private
+    def thirty_day_discharge_average_per_hundred
+      AveragePresenter.new(thirty_day_discharge_average).average_per_hundred
+    end
 
-      def percentile
-        (number*100).round(0)
-      end
+    def thirty_day_discharge_average_per_hundred_status
+      AveragePresenter.new(thirty_day_discharge_average).alert_status
     end
 
+    private
+
     def reserved_ratio
       data.in_care.to_f/(data.funded + data.reserve + data.activated)
     end
 
     def activation_ratio
       data.in_care.to_f/(data.funded + data.activated)
     end
+
+    def seven_day_discharge_average
+      1
+      # average discharge / average in-care
+    end
+
+    def thirty_day_discharge_average
+      1
+    end
```

Comment:
No, they are not in use. I did a sloppy extraction and removed them in a later commit. These methods are all being delegated to another object now.